lib/JITServer/JITServer.cpp

```diff
@@ -397,7 +397,7 @@ ServerNewInterpreterThunkBlock(
 
         // REVIEW: OOP JIT should we clear arena at end?
         ArenaAllocator * arena = scriptContext->GetSourceCodeArena();
-        BYTE * localBuffer = AnewArray(arena, BYTE, bufferSize);
+        BYTE * localBuffer = HeapNewArray(BYTE, bufferSize);
```

Comment:
it's just a page. i'll leave in arena memory though... wasn't really reason to change that